zetaclient/chains/bitcoin/bitcoin.go

```diff
@@ -224,3 +216,55 @@ func (b *Bitcoin) outboundLogger(id string) *zerolog.Logger {
 
 	return &l
 }
+
+// inboundSkipper returns a skipper function for inbound observation.
+func inboundSkipper(b *Bitcoin, app *zctx.AppContext) bool {
```

Comment:
name of the function confuse, function name should be action, not noun,

like `checkSkipInbound`, etc...

Same comments for all other function name

Reply:
This follows the function type `scheduler.Skipper`. Since the function return a `bool`, can I rename it as `shouldSkipInbound`?